ActindoWorkFlow/Overview/04_WorkflowProcessElements.md

```diff
@@ -2,29 +2,30 @@
 
 ![Workflow](../../Assets/Screenshots/ActindoWorkFlow/Workflows/Workflow.png "[Workflow]")
 
-The *Workflows* module allows customers to model their own processes to match their companies structure. A workflow consists of a number of fixed elements that are chained together, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs.    
-For detailed information, see [Manage the workflows](../Operation/01_ManageWorkflows.md).
+The *Process Orchestration* module allows you to model your own processes to match your business processes. A workflow consists of a number of fixed elements that are chained together, each of them having specific characteristics. These elements are used to design the workflow, which can be edited as necessary according to customer needs.    
 
-A workflow runs from a fixed start place to a fixed end place, which can be in turn linked to another workflow. A workflow must contain at least one transition between the start and the end place. The transitions are linked through places with each other, connecting the corresponding transition input and output ports. All these elements are described more in detail below:
+For detailed information, see [Manage the workflows](../Operation/01_ManageWorkflows.md). For a detailed step-by-step description of how to use the workflow objects presented below in practice, see [Prepare the Workflows](../../POS/Integration/09_WorkflowAdjustments.md) in the *POS* documentation.
```

Comment:
Nicht, dass ich auf dein POS-Doku nicht verweisen möchte, aber ich finde, inhaltlich gehört das hierhin, in Process Orchestration-Modul. Ich fänd es komisch, als Leser, dass ich zu POS für eine Erklärung von Workflows-Elemente verwiesen werde. Eher andersrum...